docs/configuration/resource-recommender.rst

```diff
@@ -0,0 +1,126 @@
+Cost Savings (KRR)
+************************************************************
+
+Robustas `KRR <https://github.com/robusta-dev/krr>`_ is a CLI tool for optimizing resource allocation in Kubernetes clusters.
+It gathers pod usage data from Prometheus and recommends requests and limits for CPU and memory.
+This reduces costs and improves performance.
+By default, every instance of Robusta that's connected to the UI will run a KRR scan on startup. Further KRR scans can be triggered in the UI, and all scans can be viewed there.
+
+With or without the UI, you can configure additional scans on a :ref:`schedule <Scheduled>`.
+The results can be sent as a PDF to Slack or to the Robusta UI.
+
+.. grid:: 1 1 1 1
+
+    .. grid-item::
+
+        .. md-tab-set::
+
+            .. md-tab-item:: Slack
+
+                .. image:: /images/krr_slack_example.png
+                    :width: 1000px
+
+            .. md-tab-item:: Robusta UI
+
+                .. image:: /images/krr_example.png
+                    :width: 1000px
+
+
+.. robusta-action:: playbooks.robusta_playbooks.krr.krr_scan on_schedule
+
+    You can trigger a KRR scan at any time, by running the following command:
+
+    .. code-block:: bash
+
+        robusta playbooks trigger krr_scan
+
+
+Taints, Tolerations and NodeSelectors
```

Comment:
Here too, see if you can touch on the motivation in the title - e.g. "Running KRR on Specific Nodes"